From de08372ed4ba569b215d5a029547b6b0d13ed0ac Mon Sep 17 00:00:00 2001
From: Salini Venate <salini.venate@intel.com>
Date: Tue, 27 Jun 2023 09:41:07 +0530
Subject: [PATCH 3/3] Enable X86_SGX config

Kernel image creation was failing with below error:
ld.lld: error: version script assignment of 'LINUX_2.6' to symbol
'__vdso_sgx_enter_enclave' failed: symbol not defined.

Enabling X86_SGX config

Tracked-On: NA
Signed-off-by: Salini Venate <salini.venate@intel.com>
---
 .../gmin64/config-lts/linux-intel-lts2021/x86_64_defconfig      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/groups/kernel/gmin64/config-lts/linux-intel-lts2021/x86_64_defconfig b/groups/kernel/gmin64/config-lts/linux-intel-lts2021/x86_64_defconfig
index 34b4f49..ccf5f36 100644
--- a/groups/kernel/gmin64/config-lts/linux-intel-lts2021/x86_64_defconfig
+++ b/groups/kernel/gmin64/config-lts/linux-intel-lts2021/x86_64_defconfig
@@ -427,7 +427,7 @@ CONFIG_X86_INTEL_MEMORY_PROTECTION_KEYS=y
 CONFIG_X86_INTEL_TSX_MODE_OFF=y
 # CONFIG_X86_INTEL_TSX_MODE_ON is not set
 # CONFIG_X86_INTEL_TSX_MODE_AUTO is not set
-# CONFIG_X86_SGX is not set
+CONFIG_X86_SGX=y
 CONFIG_EFI=y
 # CONFIG_HZ_100 is not set
 # CONFIG_HZ_250 is not set
-- 
2.17.1

